data/data/powervs/variables-powervs.tf

```diff
@@ -46,6 +46,45 @@ variable "cloud_instance_id" {
   default     = ""
 }
 
+################################################################
+# Configure storage
+################################################################
+variable "cos_resource_group_name" {
+  type = string
+  description = "The name of the COS resource group that holds your resource"
+  default = ""
+}
+
+variable "cos_instance_name" {
+    type        = string
+    description = "The name of the COS instance that will hold your bucket"
+    default     = ""
+}
+
+variable "cos_instance_plan" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "standard"
+}
+
+variable "cos_instance_location" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "global"
+}
+
+variable "cos_storage_class" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "standard"
```

Comment:
I prefer `smart` over standard, to save some $$$$. I know it is short-lived but better than a standard plan.

explanation: `Smart Tier automatically gives you the lowest storage rate based on your monthly activity.` from docs

Reply:
1) Removing cos_instance_plan for now as it was only used to create the COS instance. If we do wind up creating the COS instance in terraform, will hardcode to standard.
2) Done
3) Done